lua_examples/init.lua

```diff
@@ -0,0 +1,34 @@
+-- init.lua example, with wifi portal
```

Comment:
This should IMO go into a sub directory like e.g. `lua_examples/eus/init.lua`

Reply:
I agree that this init.lua is "eus" specific. I have no problem moving it to eus. However, maybe we could group all examples of init.lua together... one with eus, one for waking up after deep sleep, other situations.... All I know is that when we need those examples, they are scattered around the docs, so a directory "starting up" might also work. Its up to you, I'll follow your suggeations and update the PR.

Reply:
> maybe we could group all examples of init.lua together

Good idea! Do you have a naming proposal? A directory called `init_lua` and then files like e.g. `eus-init.lua`?

Reply:
@blobule still interested in following up on this?